testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java

```diff
@@ -338,19 +348,24 @@ protected VMRunnerConfiguration createVMRunner(
     }
 
     Version testngVer = getRuntimeTestNGVersion(classpathList);
-    if (testngVer != null) {
-      if (compareVersion(testngVer, minTestNGVer) < 0) {
-        throw new CoreException(TestNGPlugin.createError(ResourceUtil.getString(
-            "TestNGLaunchConfigurationDelegate.error.testngVersionUnsupported")));
-      }
-      if (compareVersion(testngVer, new Version("6.9.0")) < 0) {
-        // add remote testng jar file at the very begining of classpath
-        // to make sure this is loaded in prior of the user's testng.jar,
-        // this applies for testng version prior to 6.9 only,
-        // since testng-remote does not compatible with 6.9.x yet
-        classpathList.add(0, BuildPathSupport.getRemoteTestNGLibPath());
-      }
+    if (testngVer == null) {

```

Comment:
from now on, we always add testng-remote jars to TestNG process, the testng-remote jar version is depends on the runtime testng version.